README.md

```diff
@@ -35,6 +35,22 @@ To use N-API in a native module:
 #include "napi.h"
 ```
 
+To make the migration to node-api easier, we have provided a script to help complete the steps listed above. To use the conversion script:
```

Comment:
Add a `## Conversion tool` header here to separate this from the previous section.